src/main/java/com/example/sample/SecurityConfiguration.java

```diff
@@ -34,8 +34,8 @@ public SecurityFilterChain configure(HttpSecurity http) throws Exception {
             .anyRequest().authenticated());
         http.logout((logout) -> logout.logoutSuccessHandler(oidcLogoutSuccessHandler()));
         // enable OAuth2/OIDC
-        http.oauth2Login();
-        http.oauth2ResourceServer().jwt();
+        http.oauth2Login(Customizer.withDefaults());
```

Comment:
I'd recommend doing a static import here and changing this to `withDefaults()`.

Reply:
You're welcome to update it if you would like; I thought I would open a quick pr since I had updated this for my own purposes, but I'm not intending to spend any more time on it.